File: find_lines.js
const fs = require('fs');

const filePath = 'c:\\Users\\gorke\\OneDrive\\Desktop\\word-game\\kelimeler.txt';

try {
    // Try reading as UTF-8 first
    const content = fs.readFileSync(filePath, 'utf8');
    const lines = content.split(/\r?\n/);
    console.log("Read successful. Total lines:", lines.length);
    
    let count = 0;
    lines.forEach((line, index) => {
        if (line.includes('| (') && line.trim().endsWith(')')) {
            console.log(`Line ${index + 1}: ${line.trim()}`);
            count++;
        }
    });
    console.log("Total matches found:", count);
} catch (err) {
    console.error("Error reading file:", err);
}

File: find_lines.py
import os

file_path = r'c:\Users\gorke\OneDrive\Desktop\word-game\kelimeler.txt'

encodings = ['utf-8', 'utf-16', 'utf-16-le', 'utf-16-be', 'latin-1', 'cp1254']

for enc in encodings:
    try:
        with open(file_path, 'r', encoding=enc) as f:
            lines = f.readlines()
            print(f"Testing encoding: {enc}")
            count = 0
            for i, line in enumerate(lines):
                if '| (' in line and line.strip().endswith(')'):
                    print(f"Line {i+1}: {line.strip()}")
                    count += 1
                if count >= 20:
                    break
            if count > 0:
                print(f"Found {count} matches with {enc}")
                break
    except Exception as e:
        print(f"Failed with {enc}: {e}")
